driver/annotations.md

```diff
@@ -0,0 +1,224 @@
+# Adding UAST Annotations
+
+# Intro
+
+Once you have written the [native part of a programming language's
+driver](driver/sdk.md) the
+next step is to write the annotation code. This is the Go language code that will
+establish the rules to transform the native driver AST into the UAST and the
+implementation uses two objects, the ObjectToNoder and the AnnotationRules on two
+separate files.  Most of the files related to the Go part of the driver are
+auto-generated by the [`bblfsh-sdk init`
+tool](driver/sdk.md), but in
+order to translate the native AST to the UAST you need to complete two skeleton
+files that are referenced in the (auto-generated) driver/main.go file.
+
+## File: parser.go
+
+### ObjectToNoder struct
```

Comment:
This would be actually the `ToNoder`. `ObjectToNoder` is just a possible implementation.